lib/image_helper.rb

```diff
@@ -2,6 +2,7 @@
 require 'fileutils'
 require 'rmagick'
 require 'zip'
+require 'shellwords'
```

Comment:
This is the first time that we have used `shellwords`.  It looks like a useful gem, but I hate to add a new dependency for a single line.  We should use shellwords for other filename escaping, at least in this file.